src/commonMain/resources/db/migration/V40__Bookmarks.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TYPE public.playlistType AS ENUM
+(
+    'Public',
+    'Private',
+    'System'
+);
+
+ALTER TYPE public.playlistType OWNER TO beatmaps;
+
+ALTER TABLE public.playlist
+ADD COLUMN type playlistType NOT NULL DEFAULT 'Public';
+
+UPDATE public.playlist
+SET type = 'Public'
+WHERE public;
+
+UPDATE public.playlist
+SET type = 'Private'
+WHERE NOT public;
+
+ALTER TABLE public.playlist
+DROP COLUMN public;
```

Comment:
In an ideal world we'd probably leave this column initially in case there was some problem that meant we wanted to roll back. Big points for the no-op update above too setting the type to the default value again 😛